docs/design/marketing-offer-program.md

```diff
@@ -0,0 +1,250 @@
+  - Copy the static link associated to their account with the default invite message from their dashboard 
+  - Twitter Share
+
+- When a referral link is accepted:
+  The invitee can accept their invite to redeem their Referral Offer by going through the Referral Link to create an account on the Satellite they were invited to/through.
+  Upon creating their account they must enter their payment information into the Stripe integration and accept/acknowledge that:
+    If they use up their referral credit before the expiration date they should expect a bill for any amount of activity they acquire beyond what was credited to them
+    When their offer period expires they should expect a bill for any activity on the Tardigrade network going forward billed monthly at unit per hour
+
+### AWARDED CREDIT
+
+- When free credit is awarded:
+It is displayed as the USD value that is equivalent to the storage & bandwidth allotted amount for the credit
+The credit is automatically applied to the account and will have a max limit that can be reached within a set timeframe, as well as an expiration date for it to be used by.
+  The user will be charged for activity that is beyond the Free Credit limit within the set timeframe
+  The user will be charged for activity that is after the expiration date.
+  Example statement to the user: 
+  “You will not be charged until all of your $XX credit is used or it expires. Your $XX credit will be applied immediately”
+  A user's free credit can only be applied to projects that they create
+
+  A client can see their awarded Free Credits in their dashboard with associated expiration date
+  A client who has been awarded storage credit for their referrals is able to retain the awarded credits when the referral offer they received the credit under has been updated/changed
+
+### CATEGORIES OF FREE CREDIT & OFFER TYPES
+
+**Default Free Tier Offer:**
+  - This is the credit that all new users who join the network receive if they create an account without a Referral Link. Once a new user successfully creates their account they will have 14 days to use their Free Credit allowance.  
+  The Default Offer will remain constant, with no expiration date to the offer and does not have a redemption cap
+  The Offer Program Admin is able to adjust the default offer
+  One Default Offer per Tardigrade Satellite.
+  When there is a lapse of the Referral Testing Offer/the Referral Testing Offer has expired or reached a cap, the Default Offer will be the default to the Referral Testing Offer that new users will redeem until a new Referral Testing Offer is generated.
+
+**Referral Testing Offer**:
+  - This is the credit awarded to the Invitee upon creating their account using a Referral Link. This credit can have a different expiration date than the awarded Referrer Credit and the Default Free Credit.
+  An admin of the Referral Program is able to easily adjust the current offer through the  Admin Satellite GUI
```

Comment:
Let's say we have Alice, Bob, and Carol. Alice is the Satellite Admin, Bob has an account, and Carol is signing up.

1. Alice configures the Satellite to have a Referral Testing Offer credit of $100.
2. Bob creates a referral link and sends it to Carol. In the email, Bob says "this referral link is for $100!"
3. Alice decides that she was way too generous and changes the credit to $10.
4. Carol eagerly signs up.

Should Carol have $10 or $100 in credits?

Reply:
I see the problem here. I think we will only allow update to be done on offers that have `NoStatus` as the Status field

Reply:
the other option is to include the offer "version" in the referral link? so when an offer is edited, there is a new version, and referral links are updated? 

i suppose the downside of that is that bob could continue happily sharing his $100 referral link with abandon, so perhaps there's problems both ways and this is a good tradeoff. 

summary: i like your solution

Reply:
Yes, I think the marketing team would like to keep the referral link static so sadly introducing versioning would not meet that requirement